Classes/Parser/ScssParser.php

```diff
@@ -83,9 +85,36 @@ protected function parseFile($file, $settings)
                 'sourceMapBasepath' => '<PATH DOES NOT EXIST BUT SUPRESSES WARNINGS>'
             ]);
         }
+        $absoluteFilename = GeneralUtility::getFileAbsFileName($file);
+        // Adds visual directory path of the initial file as import path
+        // This scenarios happens, when e.g. developing packages using the `path`
+        // repository feature of Composer - having one package in `public/typo3conf/ext/`
+        // and the other one symlinked in e.g. `packages/`.
+        // Since the PHP SCSS parser works on resolved real paths, the symlinked context is lost.
+        $visualImportPath = dirname(GeneralUtility::getFileAbsFileName($file));
```

Comment:
`$absoluteFilename` can be used here

Reply:
Yeah, saw this too. Thinking also about to rename $visualImportPath because I'm using this variable too for my patch #868 and in this context the naming is wrong. But I thought about streamlining this after the merge of this two patches...